views/components/user-navigation/macro.njk

```diff
@@ -13,26 +13,12 @@
                     {% if previousPage %}
                         <li class="user-nav__pagination-link user-nav__pagination-link--prev">
                             <span class="user-nav__pagination-arrow">←</span>
-                            <a href="{{ previousPage.url }}">
-                                <span>
-                                    {{ __('applyNext.navigation.previous') }}
-                                </span>
-                                <span class="user-nav__pagination-subtitle">
-                                    {{ previousPage.label }}
-                                </span>
-                            </a>
+                            <input class="btn-link" name="previousBtn" type="submit" value="{{ __('applyNext.navigation.previous') }}"/>
                         </li>
                     {% endif %}
                     {% if nextPage %}
                         <li class="user-nav__pagination-link user-nav__pagination-link--next">
-                            <a href="{{ nextPage.url }}">
-                                <span>
-                                    {{ __('applyNext.navigation.next') }}
-                                </span>
-                                <span class="user-nav__pagination-subtitle">
-                                    {{ nextPage.label }}
-                                </span>
-                            </a>
+                            <input class="btn-link" name="nextBtn" type="submit" formaction="" value="{{ __('applyNext.navigation.next') }}"/>
```

Comment:
How come this button has a `formaction` and the other doesn't? Do we need this?

Reply:
My bad. I have to apply the `formaction` to the this button too.